miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/RealMiniApp.kt

```diff
@@ -135,6 +140,9 @@ internal class RealMiniApp(
             miniAppDownloader.updateApiClient(it)
             miniAppInfoFetcher.updateApiClient(it)
         }
+        if (newConfig.rasProjectId != null)
```

Comment:
I think, this statement is always true, so we can write `newConfig.rasProjectId.isEmpty()` instead 🤔

Reply:
i think no need for any check here.. @khairul-alam-licon, we always have `rasProjectId` to create a miniApp so its ok not to check anything here